cmd/av/pr_queue.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/aviator-co/av/internal/utils/colors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/aviator-co/av/internal/actions"
+	"github.com/aviator-co/av/internal/avgql"
+	"github.com/shurcooL/graphql"
+	"github.com/spf13/cobra"
+)
+
+var prQueueFlags struct {
+	SkipLine bool
+	Targets  string
+}
+
+var prQueueCmd = &cobra.Command{
+	Use:          "queue",
+	Short:        "queue a pull request for the current branch",
+	SilenceUsage: true,
+	Args:         cobra.NoArgs,
+	// error or reterr from emperror.dev/errors here?
```

Comment:
Doesn't reeeeally matter that much. The main benefit is that if you run with `av --debug` it will print out stack traces, but the traces would be so shallow at this point that it doesn't matter super hard.